Geoms/CMS-2017.cc

```diff
@@ -107,6 +107,62 @@ namespace
   }
 */
```

Comment:
should the commented out part with `SetupBackwardSearch_Iter5` be removed now?

Reply:
I left it there, in case @osschar needs it for future development.
But sure, I can clean them out.

Reply:
I have removed `SetupBackwardSearch_Iter5` and explained `SetupBackwardSearch_Iter0`.